modules/dnn/test/test_model.cpp

```diff
@@ -5,6 +5,7 @@
 #include "test_precomp.hpp"
 #include <opencv2/dnn/shape_utils.hpp>
 #include "npy_blob.hpp"
+
```

Comment:
> modules/dnn/test/test_model.cpp

Wrong test file is chosen.
This file is used for testing of "DNN Model API".

Darknet models should go into `test_darknet_importer.cpp` file.
If it is not related to Darknet framework, then it should got into `test_onnx_importer.cpp`.

This is how it is organized - anyway we don't need extra mess.

Reply:
Test is now moved to `test_onnx_importer.cpp`.